modules/material/themes/material/expirychecker/about2expire.twig

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="{{ currentLanguage }}">
+<head>
+  <title>{{ '{about2expire:title}'|trans }}</title>
+
+  {% include 'header.twig' %}
+</head>
+<body>
+<div class="mdl-layout mdl-layout--fixed-header fill-viewport">
+  <header class="mdl-layout__header">
+    <div class="mdl-layout__header-row">
+        <span class="mdl-layout-title">
+          {{ '{about2expire:header}'|trans }}
+        </span>
+    </div>
+  </header>
+  <main class="mdl-layout__content" layout-children="column">
+    <form layout-children="column">
+      {% for name, value in formData %}
+        <input type="hidden" name="{{ name|e }}" value="{{ value|e }}">
```

Comment:
I don't _think_ we need these `|e` here, since that same `html` escaping strategy will be automatically applied.

Reply:
This shouldn't cause a problem, though.

Reply:
It's puzzling to me, still, about what is done by default. SSP devs seem to use the |e filter. Maybe just habit, or maybe they know something I don't know.